Formula/iso-codes.rb

```diff
@@ -14,9 +15,17 @@ class IsoCodes < Formula
   end
 
   depends_on "gettext" => :build
+  depends_on :python3
+  depends_on "pkg-config" => :run
 
   def install
     system "./configure", "--prefix=#{prefix}"
+    system "make"
+    system "make", "check"
     system "make", "install"
   end
+
+  test do
+    assert_match /iso_639-2 iso_639-3 iso_639-5 iso_3166-1/, shell_output("#{Formula["pkg-config"].opt_bin}/pkg-config --variable=domains iso-codes")
```

Comment:
The current test stretches the formula out quite a bit. Just asking that we split it onto the two lines like that to make sure it isn't stretched beyond 83 characters in the test rather than, er, 145. Helps with readability.